### PR TITLE
[Storage]Add Patch Release ChangeLog

### PR DESCRIPTION
**Pull request by xiafu-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-blob/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 # Release History
 
-## 12.8.1b1 (Unreleased)
+## 12.8.1 (2021-04-20)
+**Fixes**
+- Fixed retry on large block upload 
+- Make `AccountName`, `AccountKey` etc. in conn_str case insensitive
+- Fixed downloader.chunks() return chunks in different size (#9419, #15648)
+- Enabled exists() for CPK encrypted blobs (#18041)
+- Fix service props Metrics class
```
Comment (tasherif-msft) [inline sdk/storage/azure-storage-blob/CHANGELOG.md:9]: This one is related to STG777 (it was merged to that branch)

**Review by tasherif-msft — APPROVED** [review]
besides the one comment we're good to go for the changelog

File: sdk/storage/azure-storage-queue/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Release History
 
-## 12.1.6 (Unreleased)
+## 12.1.6 (2021-04-20)
+**Fixes**
+- Make `AccountName`, `AccountKey` etc. in conn_str case insensitive
+- Fixed unclosed ThreadPoolExecutor (#8955)
+- Optimized memory usage for datalake file uploads large chunks (#16890)
```
Comment (tasherif-msft) [inline sdk/storage/azure-storage-queue/CHANGELOG.md:7]: this is datalake related?

**Review by tasherif-msft — COMMENTED** [review]
some comments were left

File: sdk/storage/azure-storage-queue/CHANGELOG.md (hunk 2 above)
Comment (xiafu-msft) [reply sdk/storage/azure-storage-queue/CHANGELOG.md:7]: hahaha read the description again, it's copied from your pr

Comment (tasherif-msft) [reply sdk/storage/azure-storage-queue/CHANGELOG.md:7]: No I mean this is a datalake change but right now its under queues changelog

Comment (xiafu-msft) [reply sdk/storage/azure-storage-queue/CHANGELOG.md:7]: hahahaha you are so right! I don't know why I paste it here

**Pull request merged**